testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentDefaults.java

```diff
@@ -15,7 +15,7 @@
 
 public final class EnvironmentDefaults
 {
-    public static final String DOCKER_IMAGES_VERSION = "41";
+    public static final String DOCKER_IMAGES_VERSION = "45";
```

Comment:
This probably should be first commit (not last), or squashed.

Reply:
nvrm, desn't have to be, given current contents of PR

Reply:
Doesn't have to be. It works without that :)